File: specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/examples-java/ElasticPoolDatabaseActivityList.java
import com.azure.core.util.Context;

/** Samples for ElasticPoolDatabaseActivities ListByElasticPool. */
public final class Main {
    /*
     * x-ms-original-file: specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01/examples/ElasticPoolDatabaseActivityList.json
     */
    /**
     * Sample code: List elastic pool database activity.
     *
     * @param azure The entry point for accessing resource management APIs in Azure.
     */
    public static void listElasticPoolDatabaseActivity(com.azure.resourcemanager.AzureResourceManager azure) {
        azure
            .sqlServers()
            .manager()
            .serviceClient()
            .getElasticPoolDatabaseActivities()
            .listByElasticPool("sqlcrudtest-4673", "sqlcrudtest-603", "7537", Context.NONE);
    }
}

File: specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/examples-python/ElasticPoolDatabaseActivityList.py
from azure.identity import DefaultAzureCredential
from azure.mgmt.sql import SqlManagementClient

"""
# PREREQUISITES
    pip install azure-identity
    pip install azure-mgmt-sql
# USAGE
    python elastic_pool_database_activity_list.py

    Before run the sample, please set the values of the client ID, tenant ID and client secret
    of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID,
    AZURE_CLIENT_SECRET. For more info about how to get the value, please see:
    https://docs.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal
"""


def main():
    client = SqlManagementClient(
        credential=DefaultAzureCredential(),
        subscription_id="9d4e2ad0-e20b-4464-9219-353bded52513",
    )

    response = client.elastic_pool_database_activities.list_by_elastic_pool(
        resource_group_name="sqlcrudtest-4673",
        server_name="sqlcrudtest-603",
        elastic_pool_name="7537",
    )
    for item in response:
        print(item)


# x-ms-original-file: specification/sql/resource-manager/Microsoft.Sql/stable/2014-04-01-legacy/examples/ElasticPoolDatabaseActivityList.json
if __name__ == "__main__":
    main()
